frontends/common/options.cpp

```diff
@@ -89,6 +89,12 @@ CompilerOptions::CompilerOptions() : Util::Options(defaultMessage) {
     registerOption("--testJson", nullptr,
                     [this](const char*) { debugJson = true; return true; },
                     "Dump and undump the IR");
+    registerOption("--pi-file", "file",
+                   [this](const char* arg) { piFile = arg; return true; },
+                   "Write a PI control plane API description to the specified file.");
+    registerOption("--pi-as-json", nullptr,
+                   [this](const char* arg) { piAsJson = true; return true; },
+                   "Write out the PI API description as human-readable JSON.");
```

Comment:
Actually I'm curious. If the purpose is to be human-readable, why not use `DebugString` on the proto? Or is it expected that the JSON output may be consumed by other tools?